src/sentry/models/savedsearch.py

```diff
@@ -18,6 +37,9 @@ class SavedSearch(Model):
     type = models.PositiveSmallIntegerField(default=SearchType.ISSUE.value, null=True)
     name = models.CharField(max_length=128)
     query = models.TextField()
+    sort = models.CharField(
+        max_length=16, default=SortOptions.DATE, choices=SortOptions.as_choices()
```

Comment:
Since this table has 3.8mm rows we can't set a default like this, since it'll lock the table for a long time while it backfills the column.

What we can do here is make the column nullable, then write a backfill script to fill in the values. We can leave the default in so that new rows have the correct value, see my comment on the migration for info on how to make that work right.

You could include the backfill script as just another step on your current migration, just make sure it's set to `atomic=False`